app/src/main/res/values/strings.xml

```diff
@@ -291,36 +291,36 @@
-    <string name="biometric_auto_open_prompt_title">Auto open biometric prompt</string>
-    <string name="biometric_auto_open_prompt_summary">Automatically open biometric prompt when a biometric key is defined for a database</string>
+    <string name="biometric_auto_open_prompt_title">Auto-open biometric prompt</string>
+    <string name="biometric_auto_open_prompt_summary">Ask for biometric if the database is set up to require it</string>
     <string name="biometric_delete_all_key_title">Delete encryption keys</string>
     <string name="biometric_delete_all_key_summary">Delete all encryption keys related to biometric recognition</string>
-    <string name="biometric_delete_all_key_warning">Are you sure you want to delete all keys related to biometric recognition?</string>
+    <string name="biometric_delete_all_key_warning">Delete all keys related to biometric recognition?</string>
     <string name="unavailable_feature_text">Could not start this feature.</string>
     <string name="unavailable_feature_version">Your Android version %1$s does not meet the minimum version %2$s required.</string>
     <string name="unavailable_feature_hardware">Could not find the corresponding hardware.</string>
     <string name="file_name">Filename</string>
     <string name="path">Path</string>
     <string name="assign_master_key">Assign a master key</string>
-    <string name="full_file_path_enable_title">File path</string>
-    <string name="full_file_path_enable_summary">View the full file path</string>
+    <string name="full_file_path_enable_title">Filepath</string>
+    <string name="full_file_path_enable_summary">View the full storage location</string>
     <string name="database_data_compression_title">Data compression</string>
     <string name="database_data_compression_summary">Data compression reduces the size of the database.</string>
     <string name="recycle_bin_title">Recycle bin usage</string>
     <string name="recycle_bin_summary">Moves groups and entries to \"Recycle bin\" group before deleting</string>
     <string name="recycle_bin_group_title">Recycle bin group</string>
-    <string name="max_history_items_title">Max. history items</string>
-    <string name="max_history_items_summary">Limit the number of history items per entry</string>
-    <string name="max_history_size_title">Max. history size</string>
-    <string name="max_history_size_summary">Limit the history size (in bytes) per entry</string>
+    <string name="max_history_items_title">History item limit</string>
+    <string name="max_history_items_summary">Set highest number of history items per entry</string>
+    <string name="max_history_size_title">History size limit</string>
+    <string name="max_history_size_summary">Set biggest history size (in bytes) per entry</string>
     <string name="settings_database_recommend_changing_master_key_title">Recommend renewal</string>
```

Comment:
Already modified in the develop branch

Reply:
Should this target develop?

Reply:
Line 317 in string.xml, git develop branch ->
https://github.com/Kunzisoft/KeePassDX/blob/develop/app/src/main/res/values/strings.xml

Reply:
This isn't 317, and neither of them are changed there?